Mathlib/Data/Multiset/Basic.lean

```diff
@@ -627,45 +644,24 @@ theorem le_iff_exists_add {s t : Multiset α} : s ≤ t ↔ ∃ u, t = s + u :=
 
 @[simp]
 theorem cons_add (a : α) (s t : Multiset α) : a ::ₘ s + t = a ::ₘ (s + t) := by
-  rw [← singleton_add, ← singleton_add, add_assoc]
+  rw [← singleton_add, ← singleton_add, Multiset.add_assoc]
 
 @[simp]
 theorem add_cons (a : α) (s t : Multiset α) : s + a ::ₘ t = a ::ₘ (s + t) := by
-  rw [add_comm, cons_add, add_comm]
+  rw [Multiset.add_comm, cons_add, Multiset.add_comm]
 
 @[simp]
 theorem mem_add {a : α} {s t : Multiset α} : a ∈ s + t ↔ a ∈ s ∨ a ∈ t :=
   Quotient.inductionOn₂ s t fun _l₁ _l₂ => mem_append
 
-theorem mem_of_mem_nsmul {a : α} {s : Multiset α} {n : ℕ} (h : a ∈ n • s) : a ∈ s := by
-  induction' n with n ih
-  · rw [zero_nsmul] at h
-    exact absurd h (not_mem_zero _)
-  · rw [succ_nsmul, mem_add] at h
-    exact h.elim ih id
-
-@[simp]
-theorem mem_nsmul {a : α} {s : Multiset α} {n : ℕ} : a ∈ n • s ↔ n ≠ 0 ∧ a ∈ s := by
-  refine ⟨fun ha ↦ ⟨?_, mem_of_mem_nsmul ha⟩, fun h => ?_⟩
-  · rintro rfl
-    simp [zero_nsmul] at ha
-  obtain ⟨n, rfl⟩ := exists_eq_succ_of_ne_zero h.1
-  rw [succ_nsmul, mem_add]
-  exact Or.inr h.2
-
-lemma mem_nsmul_of_ne_zero {a : α} {s : Multiset α} {n : ℕ} (h0 : n ≠ 0) : a ∈ n • s ↔ a ∈ s := by
-  simp [*]
-
-theorem nsmul_cons {s : Multiset α} (n : ℕ) (a : α) :
-    n • (a ::ₘ s) = n • ({a} : Multiset α) + n • s := by
-  rw [← singleton_add, nsmul_add]
+end add
 
 /-! ### Cardinality -/
 
 
 /-- The cardinality of a multiset is the sum of the multiplicities
   of all its elements, or simply the length of the underlying list. -/
-def card (s : Multiset α) : ℕ := Quot.liftOn s length fun _l₁ _l₂ => Perm.length_eq
+def card : Multiset α → ℕ := Quot.lift length fun _l₁ _l₂ => Perm.length_eq
```

Comment:
This seems unrelated, and there's more unrelated changes in the PR. Would you mind splitting them out or reverting them?

Reply:
This specific change is just that I apparently changed my mind between the two PRs